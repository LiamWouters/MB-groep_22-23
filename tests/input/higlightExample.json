{
  "This blue string has a purple value" : 123,
  "Another blue string has a pink special value": true,
  "Blue strings can also have another, green string value": "See?",
  "Now for arrays": [
    true,
    null,
    {
      "Recursion": "We conquer it like a boss.",
      "This": [
        {"a": "n"},
        {"c": [{"d": 4.0}]}
      ]
    },
    345.5,
    "filler",
    [null, 456, "more filler", "last filler"]
  ]
}