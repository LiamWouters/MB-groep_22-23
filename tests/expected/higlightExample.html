<body><pre>{
</pre><pre>  <span style="color:blue;">"This blue string has a purple value"</span> :<span style="color:deeppink;"> </span><span style="color:purple;">1</span><span style="color:purple;">2</span><span style="color:purple;">3</span>,
</pre><pre>  <span style="color:blue;">"Another blue string has a pink special value"</span>:<span style="color:deeppink;"> </span><span style="color:deeppink;">t</span><span style="color:deeppink;">r</span><span style="color:deeppink;">u</span><span style="color:deeppink;">e</span>,
</pre><pre>  <span style="color:blue;">"Blue strings can also have another, green string value"</span>:<span style="color:deeppink;"> </span><span style="color:green;">"See?"</span>,
</pre><pre>  <span style="color:blue;">"Now for arrays"</span>:<span style="color:deeppink;"> </span>[<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;">t</span><span style="color:deeppink;">r</span><span style="color:deeppink;">u</span><span style="color:deeppink;">e</span>,<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;">n</span><span style="color:deeppink;">u</span><span style="color:deeppink;">l</span><span style="color:deeppink;">l</span>,<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span>{
</pre><pre>      <span style="color:blue;">"Recursion"</span>:<span style="color:deeppink;"> </span><span style="color:green;">"We conquer it like a boss."</span>,
</pre><pre>      <span style="color:blue;">"This"</span>:<span style="color:deeppink;"> </span>[<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span>{<span style="color:blue;">"a"</span>:<span style="color:deeppink;"> </span><span style="color:green;">"n"</span>},<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span>{<span style="color:blue;">"c"</span>:<span style="color:deeppink;"> </span>[{<span style="color:blue;">"d"</span>:<span style="color:deeppink;"> </span><span style="color:purple;">4</span><span style="color:purple;">.</span><span style="color:purple;">0</span>}]}<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span>]
</pre><pre>    },<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:purple;">3</span><span style="color:purple;">4</span><span style="color:purple;">5</span><span style="color:purple;">.</span><span style="color:purple;">5</span>,<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:green;">"filler"</span>,<span style="color:deeppink;">
</span></pre><pre><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span><span style="color:deeppink;"> </span>[<span style="color:deeppink;">n</span><span style="color:deeppink;">u</span><span style="color:deeppink;">l</span><span style="color:deeppink;">l</span>,<span style="color:deeppink;"> </span><span style="color:purple;">4</span><span style="color:purple;">5</span><span style="color:purple;">6</span>,<span style="color:deeppink;"> </span><span style="color:green;">"more filler"</span>,<span style="color:deeppink;"> </span><span style="color:green;">"last filler"</span>]
</pre><pre>  ]
</pre><pre>}</pre></body>